src/components/ComponentParts/RedButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  link: string;
+  buttonMessage: string;
+  buttonType: 'submit' | 'reset' | 'button' | 'menu';
+  onClick?: () => void;
+}
+
+export const RedButton = ({ link, buttonType, buttonMessage, onClick }: Props) =>
+  buttonType === 'submit' ? (
+    <button
+      type="submit"
+      className="transition ease-in-out delay-100 hover:-translate-y-1 hover:scale-105 duration-200 text-primary-font-color font-thin text-md py-2 px-4 tracking-wider bg-login-btn-color"
```

Comment:
Wiem, że to nie Twoje, ale nazwa `login-btn-color` to jest do zmiany. Ten kolor to chyba dotyczy wszystkich przycisków? @EckhartPL ?
Warto by było spojrzeć jeszcze raz dokładnie na inne nazwy kolorów :)

Reply:
Zmienione na DefaultSubmitButton. 
DefaultButton wydaje mi się zbyt mało dokładne.